Data Base Creation

kategamo opened the pull request:
DB Creation
>What:Database creation for chat space
>Why:A database management system is necessity because it manages data efficiently and allows users to perform multiple tasks with ease.
>Points:
1.Not Null
2.Association
3.add_index
Data Base Edit2:
Edited points 
(1) Remove columns from Users table 
(2)Change the volume name from Messages table
(3)Modified column name from Members table

kategamo committed:
Data Base Creation

TOMARUMARU commented on README.md:
```diff
@@ -12,6 +12,50 @@ Things you may want to cover:
+# users table
+|Column|Type|Options|
+|------|----|-------|
+
```
nameカラムを示す記述を書きましょう！

TOMARUMARU commented on README.md:
```diff
@@ -12,6 +12,50 @@ Things you may want to cover:
+has_many :messages
+has_many :groups
+add_index :users, :name
+add_index :users, :email, unique:true
```
emailカラムはdeviseで自動生成されます！

TOMARUMARU commented on README.md:
```diff
@@ -12,6 +12,50 @@ Things you may want to cover:
+# messages table
+|Column|Type|Options|
+|------|----|-------|
+|body|text|null: false|
```
画像だけでも投稿できる仕様なので、null: falseは外しましょう！

TOMARUMARU commented on README.md:
```diff
@@ -12,6 +12,50 @@ Things you may want to cover:
+|body|text|null: false|
+|user_id|integer|null: false foreign_key: true|
+|group-id|integer|null: false foreign_key: true|
+|created_at|datetime|null: false|
```
cureated_atは自動生成されます！

TOMARUMARU commented on README.md:
```diff
@@ -12,6 +12,50 @@ Things you may want to cover:
+|------|----|-------|
+|name|string|null: false|
+|user_id|integer|null: faulse foreign_key: true|
+|created_at|datetime|null: false|
```
cureated_atは自動生成されます！

TOMARUMARU commented on README.md:
```diff
@@ -12,6 +12,50 @@ Things you may want to cover:
+|image|string|
+
+## association
+belongs_to :users
```
belongs_toは単数形です！

TOMARUMARU commented on README.md:
```diff
@@ -12,6 +12,50 @@ Things you may want to cover:
+
+## association
+belongs_to :users
+belongs_to :groups
```
belongs_toは単数形です！

TOMARUMARU commented on README.md:
```diff
@@ -12,6 +12,50 @@ Things you may want to cover:
+|Column|Type|Options|
+|------|----|-------|
+|name|string|null: false|
+|user_id|integer|null: faulse foreign_key: true|
```
user_idはmembersテーブルによって管理するので必要ありません！

TOMARUMARU commented on README.md:
```diff
@@ -12,6 +12,50 @@ Things you may want to cover:
+belongs_to :group
+belongs_to :user
 
 * Database initialization
```
ここ以下の記述は削除しましょう！

TOMARUMARU reviewed (commented):
chatspaceではuserとgroupを中間テーブル(members)によって管理します！中間テーブルのアソシエーションを調べてみてください！

kategamo committed:
read_me_edit3

kategamo committed:
read_me_edit4

kouuuki commented:
```+|name|string|null: faulse|```

こちらタイプミスがございます！

kategamo committed:
read_me_edit5

kategamo committed:
git-hub

kategamo committed:
Revert "git-hub"

This reverts commit ecb5557ebdbfee979b3472b39c5785f5e7ee4021.

kategamo committed:
Front Built

What: The front page of Chat-Space
Why: Top Page is necessity and it have to be attractive.

kategamo commented:
恐れ入りますが、DB作成時のブランチのままマークアップをコミットしてしまいました。
チャット画面のレビューお願いいたします。
